gulp/tasks/test.js

```diff
@@ -175,32 +176,53 @@ function _createPSITunnel() {
   var host = envvars.TEST_HTTP_HOST;
   var port = envvars.TEST_HTTP_PORT;
   var path = _parsePath( commandLineParams.u );
-  var url = host + ':' + port + path;
+  var url = commandLineParams.u || host + ':' + port + path;
   var strategy = commandLineParams.s || 'mobile';
 
-  // Create a promise that the server is reachable and we can create a tunnel.
-  var promise = new Promise( function( resolve, reject ) {
-    // Check if server is reachable.
-    isReachable( url, function( err, reachable ) {
-      if ( err || !reachable ) {
-        reject( url + ' is not reachable. Is your local server running?' );
-      }
-      // Create the local tunnel.
-      localtunnel( port, function( tunErr, tunnel ) {
-        url = tunnel.url + path;
-        if ( tunErr ) {
-          tunnel.close();
-          reject( 'Error creating local tunnel for PSI: ' + tunErr );
-        }
-        resolve( {
-          url:    [ url, '--strategy=' + strategy ],
-          tunnel: tunnel
-        } );
-      } );
+  /**
+   * Create local tunnel and pass promise params
+   * to callback function.
+   * @param {Boolean} reachable If port is reachable.
+   * @returns {Promise} A promise which calls local tunnel.
+   */
+  function _createLocalTunnel( url, reachable ) {
+    if ( !reachable ) {
+      return Promise.reject( url +
+        ' is not reachable. Is your local server running?'
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2940/commits/7f4c1a32af392cd0b37bb28d9a66ba0bd2483cb5